src/noisepy/seis/plotting_modules.py

```diff
@@ -690,27 +694,35 @@ def plot_all_moveout(
     indx1 = int((maxlag - disp_lag) / dt)
     indx2 = indx1 + 2 * int(disp_lag / dt) + 1
 
+    if sample_pairs < 1.0:
```

Comment:
@mdenolle @niyiyu @kuanfufeng - I'd like your input here. When testing this plot on a large number of station pairs (e.g. 40k) it can take a long time. So I added a parameter to sub-sample and only plot a portion of the available stations. Here are the times when running a notebook on AWS:
```
5% sampling -> 1.85 mins
25% sampling -> 9.4 mins
100%  (no) sampling -> 37 mins
```
This helps when debugging but I don't if it makes any sense to do so when analyzing the data. I'm happy to take it out or maybe there's a better way to sub-sample instead of random? The problem is that reading the metadata is just as expensive as reading the data.